### PR TITLE
Use the InvariantCulture to format DateTime.

### PR DESCRIPTION
**Pull request by kostub**
This allows dates in Arabic/Thai locales to be entered correctly in parse.

### CODE REVIEW

File: ParseCore/Internal/User/Controller/ParseCurrentUserController.cs [hunk 1]
```diff
@@ -46,10 +47,12 @@ public class ParseCurrentUserController : IParseCurrentUserController {
             var data = user.ServerDataToJSONObjectForSerialization();
             data["objectId"] = user.ObjectId;
             if (user.CreatedAt != null) {
-              data["createdAt"] = user.CreatedAt.Value.ToString(ParseClient.DateFormatStrings.First());
+              data["createdAt"] = user.CreatedAt.Value.ToString(ParseClient.DateFormatStrings.First(),
```
Comment (richardjrossiii) [inline ParseCore/Internal/User/Controller/ParseCurrentUserController.cs]: Fix whitespace here...

File: ParseCore/Internal/User/Controller/ParseCurrentUserController.cs [hunk 2]
```diff
@@ -46,10 +47,12 @@ public class ParseCurrentUserController : IParseCurrentUserController {
...
+                                CultureInfo.InvariantCulture);
             }
             if (user.UpdatedAt != null) {
-              data["updatedAt"] = user.UpdatedAt.Value.ToString(ParseClient.DateFormatStrings.First());
+                            data["updatedAt"] = user.UpdatedAt.Value.ToString(ParseClient.DateFormatStrings.First(),
```
Comment (richardjrossiii) [inline ParseCore/Internal/User/Controller/ParseCurrentUserController.cs]: ... and here

**Comment by richardjrossiii** [comment]
Looks good! Just fix up the whitespace, accept the CLA (hopefully the Facebook bot comes along sometime soon and pings it), and we should be good to go for merging.

**Comment by kostub** [comment]
Thanks! Fixed the whitespace.

There is another place where this bug needs to be fixed: in FacebookAuthenticationProvider.cs. However I could not find that code on github. Is that not open source?

**Comment by richardjrossiii** [comment]
@kostub Since #169, for now the social aspect of the SDK is 'gone'. It will be properly modularized at a future date. When that is done, I'll make sure to include this fix.

I just checked and noticed that you have actually signed the CLA previously, so I'll merge this commit now. Thanks!

**Pull request merged**